go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -7670,4 +7670,40 @@ var DoltTempTableScripts = []queries.ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "drop temporary table behavior",
```

Comment:
These should go in GMS

Reply:
they can't, gms doesn't support temporary tables